### ISSUE TITLE
Files to reproduce QA results of NQ-Tables test set

### ISSUE THREAD

**Issue by km9mn** [google-research/tapas#183]
Hi,
I want to reproduce the QA results of NQ-Tables test set in the paper "Open Domain Question Answering over Tables via Dense Retrieval".

As I know, I need the test_knn.jsonl file to generate retrieval results and create training data for reader model.

Can you share the file?
